iaso/models/storage.py

```diff
@@ -0,0 +1,196 @@
+# TODO: need better type annotations in this file
+import uuid
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+from iaso.models import Entity, Instance, OrgUnit, Account
+
+
+class StorageDevice(models.Model):
+    """
+    A storage device used by the mobile application
+
+    For the first use-case (current situation), storage devices are NFC cards that are linked to an entity (a person).
+
+    We'd like to keep that flexible (potentially, other storage devices type could be used for other purposes), so it's
+    important to keep this model pretty generic (business logic and specific data and logic pushed to as much as possible
+    to StorageLogEntry and to the API).
+
+    StorageDevices are linked to an account, only users from the correct account can see/use/edit them.
+    """
+
+    NFC = "NFC"
+    USB = "USB"
+    SD = "SD"
+
+    STORAGE_TYPE_CHOICES = [
+        (NFC, "NFC"),
+        (USB, "USB"),
+        (SD, "SD"),
+    ]
+
+    OK = "OK"
+    BLACKLISTED = "BLACKLISTED"
+
+    STATUS_CHOICES = [
+        (OK, "OK"),
+        (BLACKLISTED, "BLACKLISTED"),
+    ]
+
+    # Reasons a Device can be blacklisted
+    STOLEN = "STOLEN"
+    LOST = "LOST"
+    DAMAGED = "DAMAGED"
+    ABUSE = "ABUSE"
+    OTHER = "OTHER"
+
+    STATUS_REASON_CHOICES = [
+        (STOLEN, "STOLEN"),
+        (LOST, "LOST"),
+        (DAMAGED, "DAMAGED"),
+        (ABUSE, "ABUSE"),
+        (OTHER, "OTHER"),
+    ]
+
```

Comment:
Use django's models.TextChoices

It's a empowered enum especially for choice and it's more readable